extension/src/plots/vega/util.ts

```diff
@@ -294,23 +307,30 @@ export const extendVegaSpec = (
     return updatedSpec
   }
 
-  const update = getSpecEncodingUpdate(encoding)
+  const update = getSpecEncodingUpdate(updatedSpec, encoding)
 
   return mergeUpdate(updatedSpec, update)
 }
 
 export const reverseOfLegendSuppressionUpdate = () => ({
   spec: {
     encoding: {
-      color: { legend: { disable: false } },
-      shape: {
+      color: {
         legend: {
           disable: false
         }
       },
+      shape: {
+        legend: {
+          disable: false,
+          symbolFillColor: 'grey'
+        }
+      },
       strokeDash: {
         legend: {
-          disable: false
+          disable: false,
+          symbolFillColor: 'transparent',
+          symbolStrokeColor: 'grey'
```

Comment:
[F] Moved these as per the ask in #2452. Thought that the approach that I was going to take was going to be significantly different to this one.